pyproject.toml

```diff
@@ -87,6 +87,7 @@ viz = [
 
 [project.scripts]
 ansys-mechanical = "ansys.mechanical.core.run:cli"
+ansys-mechanical-autocomplete = "ansys.mechanical.core.autocomplete:cli"
```

Comment:
I'm not sure about the entrypoint name. What about
`ansys-mechanical-ideconfig`? (Don't love this either but I think it's more clear)

Reply:
Sure, I think that makes more sense too